wordpress-mobile/WordPress-Android
# App crashes on changing the password

Fixes #16526

This PR fixes below crash on changing account password:

```
java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.String java.lang.CharSequence.toString()' on a null object reference
at org.wordpress.android.ui.prefs.AccountSettingsFragment.onPreferenceChange(AccountSettingsFragment.java:197)
at android.preference.Preference.callChangeListener(Preference.java:1254)
```

To test:

1. Log in with a WordPress.com account.
2. Head to Me > Account Settings and tap "Change password".
3. Enter a new password and tap "Ok" to change it.
4. Note that the application doesn't crash and the password is changed.

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
